### PR TITLE
Fix previous and next buttons for blog posts

### PR DESCRIPTION
**Pull request by erikjalevik**
Each blog post page now makes extra queries to figure out the links to use for the next/previous buttons at the bottom, so that they always work.

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit 4e6a5e2c9947edfa3d93b2875acf7435f23c62cf

https://deploy-preview-217--masifunde.netlify.com

File: styling/theme.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 export default {
   black: '#4F463F',
+  whiteAlpha: '#FFFFFF80',
```
Comment (sarunast) [inline styling/theme.js]: That is pretty cool I didn't know that there was an option to do this. However, it looks like this is a new feature https://caniuse.com/#feat=css-rrggbbaa. It is not supported by all IE browsers. Probably better to fall back to `RGBA`. 

Or you could define it as a HEX colour code but convert it javascript function to `RGB` and then add an `A`lpha channel on your own.

File: styling/theme.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
   orange: '#FE9933',
   orangeRed: '#FF621D',
+  orangeRedAlpha: '#FF621D80',
```
Comment (sarunast) [inline styling/theme.js]: Same issue as in previous color code issue.

File: styling/theme.js [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
   orangeBackgroundLight: '#faf2e6',
   orangeBackgroundMedium: '#f2dac9',
   orangeBackgroundDark: '#f1ccb0',
   green: '#17DD73',
+  greenAlpha: '#17DD7380',
```
Comment (sarunast) [inline styling/theme.js]: Same issue as in previous color code issue.

File: styling/theme.js (hunk 1 above)
Comment (erikjalevik) [reply styling/theme.js]: Oh, I thought that was age-old syntax. Will fix.

**Pull request merged**